tests/functional/utils.py

```diff
@@ -43,7 +49,7 @@ def get_run_result(secrets, run_id):
     return run['conclusion']
 
 
-@retry(stop_max_delay=15_000)
+@retry(stop_max_delay=15_000, wait_fixed=1000)
```

Comment:
Hit GitHub request limit while testing, set this interval period to slow down the requests.